src/App.js

```diff
@@ -1,6 +1,76 @@
 import React from 'react';
-import './App.css';
+import './app.scss';
+import { Pagination } from './components/Pagination/Pagination';
 
-const App = () => <h1>Pagination</h1>;
+const perPageOptions = [3, 5, 10, 20, 50];
+
+class App extends React.PureComponent {
+  state = {
+    currentPage: 9,
+    perPage: 5,
+    total: 42,
+  }
+
+  onPerPageChange = (event) => {
+    this.setState({
+      perPage: +event.target.value,
+      currentPage: 1,
+    });
+  }
+
+  onPageChange = (page) => {
+    this.setState({
+      currentPage: page,
+    });
+  }
+
+  nextPage = () => {
+    this.setState(state => ({
+      currentPage: state.currentPage + 1,
+    }));
+  }
+
+  prevPage = () => {
+    this.setState(state => ({
+      currentPage: state.currentPage - 1,
+    }));
+  }
```

Comment:
pls rename these functions, so they contain an action
mb moveToNextPage and moveToPrevPage